### ISSUE TITLE
Fix subscript and superscript display for variables that have both

### ISSUE THREAD

**Issue by smiths** [JacquesCarette/Drasil#3188]
When we have a variable with a both subscript and a superscript it displays in awkward way.  The [symbols in Projectile](https://jacquescarette.github.io/Drasil/examples/projectile/SRS/srs/Projectile_SRS.html#Sec:ToS) are an example.  For instance, we have `a_y^c` (y-component of constant acceleration).  In both LaTeX and html the display is a_y with the superscript c.  Instead of a with the subscript y and the superscript c.  In pdf generated by LaTeX the difference looks like this:

![image](https://user-images.githubusercontent.com/1422000/208506880-36941368-e043-4b15-9408-964e9223b454.png)

The Drasil-generated LaTeX code is:

`${{a_{\text{y}}}^{\text{c}}}$`

what we want is:

`${a_{\text{y}}^{\text{c}}}$`

Hopefully, this is an easy fix, but I wonder if it was done the "wrong" way because the "right" way isn't intuitive?

We can function with the examples as they are, so I'll set this as an enhancement.

**Comment by tingyuw** [comment]
Not really the same case but this issue reminds me of this #2815. The symbols display awkwardly as well if we have two subscripts or two superscripts. Maybe can take both cases into consideration when making an enhancement.

**Comment by JacquesCarette** [comment]
Hmm, finding what code puts out an extra set of `{}` might not be so easy! Normally they help and don't hinder -- except here. I'm quite sure that this fix will require actual extra logic, not just a small code fix.

**Comment by smiths** [comment]
Yes, I was afraid that this wouldn't be minor.  We'll have to identify the problematic case, and not break all the other cases where we want the extra set of `{}`.
